siliconcompiler/remote/client.py

```diff
@@ -476,21 +489,26 @@ def _request_remote_run(chip):
     # part of the HTTP spec, so we need to manually follow the trail.
     post_params = {
         'chip_cfg': chip.schema.cfg,
-        'params': __build_post_params(chip)
+        'params': __build_post_params(chip,
+                                      job_hash=chip.get('record', 'remoteid'))
```

Comment:
Is there a reason to keep `job_hash` still around?
Instead of renaming that to `remotid` as well?

Reply:
We should rename it to `remoteid` - that will be a separate change, since the "job_hash" string is also baked into the API specs.

Reply:
we have a ticket open to correct the naming overall

Reply:
SC-615